docs/partials/apiExamples.md

```diff
@@ -3,104 +3,20 @@
 
 ## API Examples
 
-### Basic
-
-<div class="twoColDemoRow">
-  <div>
-    <div class="exampleWrapper">
-      <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basic.html) -->
-      <!-- AURO-GENERATED-CONTENT:END -->
-    </div>
-<auro-accordion lowProfile justifyRight>
-  <span slot="trigger">See code</span>
-
-<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/basic.html) -->
-<!-- AURO-GENERATED-CONTENT:END -->
-
-</auro-accordion>
-
 ### Attribute Examples
 
-#### `attributeName`
-
-Explanation and use description.
-
-<div class="exampleWrapper">
-  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basic.html) -->
-  <!-- AURO-GENERATED-CONTENT:END -->
-</div>
-<auro-accordion lowProfile justifyRight>
-  <span slot="trigger">See code</span>
-
-<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/basic.html) -->
-<!-- AURO-GENERATED-CONTENT:END -->
-
-</auro-accordion>
+#### Type
+The `type` attribute determines the type of divider to be used to seperate all of the `auro-hyperlinks`.
 
-### Property Examples
+##### Breadcrumb
 
-#### `propertyName`
-
-Explanation and use description.
+The default example when the `type=breadcrumb` attribute is present.
```

Comment:
```suggestion
Example when `auro-nav` slot is populated with `auro-nav-breadcrumb` elements.
```

Let's also add language explaining that the last breadcrumb should be the current page and that they should *not* populate the HREF attribute.